crates/adapters/src/controller/stats.rs

```diff
@@ -846,16 +866,29 @@ pub struct InputEndpointStatus {
 
     /// Whether this input endpoint is [fault tolerant](crate#fault-tolerance).
     pub is_fault_tolerant: bool,
+
+    /// Endpoint has been paused by the user.
+    ///
+    /// When `true`, the endpoint doesn't produce any data even when the pipeline
+    /// is running.
+    ///
+    /// This flag is set to `true` on startup if the `pause_on_startup` flag in the
```

Comment:
As an end user, a desired state of "paused: true/false" in the connector state would be easier to think about than a "paused_on_startup: true/false".